core/templates/core/main-feature.html

```diff
@@ -1,8 +1,7 @@
 {% load static core_tags wagtailcore_tags wagtailimages_tags %}
 
 <div class="jumbotron main-feature" style="height: 400px;">
-
-    {% image self.typed_featured_item.main_image original as background_image %}
+    {% image self.typed_featured_item.main_image fill-2048x1365 as background_image %}
     <div class="feature-image" style="background-image: url(' {{ background_image.url }} ')">
```

Comment:
The background image url probably shouldn't have spaces around it inside the quotes.